bears/python/PyFlakesBear.py

```diff
@@ -0,0 +1,29 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+@linter(executable='pyflakes',
+        output_format='regex',
+        output_regex=r'(?P<filename>.*):(?P<line>.*):\s*'
+                     r'(?P<severity>)\s(?P<message>.*)',
+        severity_map={
+            '': RESULT_SEVERITY.INFO
+        })
+class PyFlakesBear:
+    """
+    Checks Python files for logistic errors using ``pyflakes``.
```

Comment:
http://blog.sideci.com/2015/12/14/style-guide-of-python-and-lint-tool/ says logistic errors. Should I simply make it errors then?